lib/scrivener/paginater/ecto/query.ex

```diff
@@ -17,7 +17,10 @@ defimpl Scrivener.Paginater, for: Ecto.Query do
       Keyword.get_lazy(options, :total_entries, fn -> total_entries(query, repo, caller) end)
 
     total_pages = total_pages(total_entries, page_size)
-    page_number = min(total_pages, page_number)
+    allow_out_of_range_pages = Keyword.get(options, :allow_out_of_range_pages, false)
```

Comment:
Can we rename this to `allow_overflow_page_number` please?

Reply:
Done